challenge.rb

```diff
@@ -1,26 +1,25 @@
-
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |i| i.capitalize}
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |i| i == "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject { |i| i == "dog" }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.count { |i| i }
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.first(2)
 end
 
 def fetch_CD_animals(input)
-  #implement your solution here
+  input.find_all { |i| i == "cat" || i == "dog" }
```

Comment:
Think about the name of the method, and how your implementation might confuse another developer using it. :)